manifest_staging/deployment/vault-csi-provider.yaml

```diff
@@ -126,6 +126,6 @@ spec:
       volumes:
         - name: providervol
           hostPath:
-            path: "/etc/kubernetes/secrets-store-csi-providers"
+            path: "/var/run/secrets-store-csi-providers"
```

Comment:
"/etc/kubernetes/secrets-store-csi-providers" is the old provider volume folder, and "/var/run/secrets-store-csi-providers" has been the [default for a while](https://github.com/kubernetes-sigs/secrets-store-csi-driver/issues/870). Sometimes the `/etc` one is set as an additional provider volume for backwards compatibility, but not always (like with the CSI Driver operator in OpenShift).